treeherder/model/models.py

```diff
@@ -26,6 +27,10 @@
 
 logger = logging.getLogger(__name__)
 
+# MySQL Full Text Search operators, based on:
+# https://dev.mysql.com/doc/refman/8.0/en/fulltext-boolean.html
```

Comment:
We're using MySQL 5.7 at the moment, so technically the URL is https://dev.mysql.com/doc/refman/5.7/en/fulltext-boolean.html but the list of operators is the same, so doesn't matter too much.

Reply:
Oops, good catch, I'd rather keep the URL in sync with our current DB version just in case!